.github/workflows/testing-stable.yml

```diff
@@ -0,0 +1,30 @@
+name: test stable
+
+on:
+  push:
+    branches:
+    - stable
+  pull_request:
+    branches:
+    - stable
+
+jobs:     
+  test-develop:
+    uses: 
+      ./.github/workflows/testing.yml
+    with:
+      xdist: no
+      branch_name: stable
+      event_name: ${{ github.event_name }}
+    secrets: inherit
+
+  test-develop-xdist:
+    uses: 
```

Comment:
also test-stable-xdist

or we can make this more generic between both definition files, by using

test-seriell
test-parallel